src/main/sass/_page-navigation.scss

```diff
@@ -0,0 +1,105 @@
+.previous-next-navigation {
```

Comment:
maybe steal from https://github.com/alphagov/static/blob/master/app/assets/stylesheets/govuk-component/_previous-and-next-navigation.scss ?